features/admin.users.v1/components/wizard/steps/add-user-basic.tsx

```diff
@@ -126,13 +174,62 @@ export const AddUserUpdated: React.FunctionComponent<AddUserProps> = (
     const [ isFirstNameRequired, setFirstNameRequired ] = useState<boolean>(true);
     const [ isLastNameRequired, setLastNameRequired ] = useState<boolean>(true);
     const [ isEmailRequired, setEmailRequired ] = useState<boolean>(false);
+    const [ profileSchema, setProfileSchema ] = useState<ProfileSchemaInterface[]>();
+    const [ multiValuedInputFieldValue, setMultiValuedInputFieldValue ] = useState<Record<string, string>>({});
+    const [ multiValuedAttributeValues, setMultiValuedAttributeValues ] =
+        useState<Record<string, string[]>>({});
+    const [ primaryValues, setPrimaryValues ] = useState<Record<string, string>>({});
+    const [ isMultiValuedItemInvalid, setIsMultiValuedItemInvalid ] =  useState<Record<string, boolean>>({});
+    const [ simpleMultiValuedExtendedProfileSchema, setSimpleMultiValuedExtendedProfileSchema ]
+        = useState<ProfileSchemaInterface[]>();
+    const [ profileInfo, setProfileInfo ] = useState(new Map<string, string>());
 
     const formBottomRef: MutableRefObject<HTMLDivElement> = useRef<HTMLDivElement>();
     const emailRef: MutableRefObject<HTMLDivElement> = useRef<HTMLDivElement>();
 
-    const { t } = useTranslation();
+    /**
+     * This will add role attribute to countries search input to prevent autofill suggestions.
+     */
+    const onCountryRefChange: any = useCallback((node: any) => {
+        if (node !== null) {
+            node.children[0].children[1].children[0].role = "presentation";
+        }
+    }, []);
 
-    const { UIConfig } = useUIConfig();
+    const countryList: DropdownItemProps[] = CommonUtils.getCountryList();
```

Comment:
Should we memoize this value?

Reply:
Fixed with c800e90c03903255034012c6f27f8ee16d6207ae